src/components/Accordion/styles/Label.js

```diff
@@ -3,8 +3,8 @@ import get from 'extensions/themeGet';
 import userSpacing from 'extensions/userSpacing';
 
 const AccordionLabel = styled.div`
-  padding: ${userSpacing.text};
-  color: ${get('colors.primary.default')};
+  padding: 30px;
```

Comment:
`userSpacing.text` actually takes into account the cap-height of the font to provide an accurate spacing from the real text. I don't think it should be removed, it probably just needs to be configured.